BitWatch/HomeViewController.swift

```diff
@@ -10,84 +10,124 @@ import UIKit
 import BitWatchKit
 import iAd
 
-// Sizes
-var height: CGFloat = CGFloat();
```

Comment:
All of these "Globals" could have been added after 
Line 40 class HomeViewController: UIViewController, ADBannerViewDelegate {

you just had to add a ! or a ? to mark them as optional vars. Then they would have became class properties and been totally acceptable. Also, ';' aren't necessary in swift. They don't hurt anything either.